src/components/Notifications.tsx

```diff
@@ -1,56 +1,47 @@
-import { useMemo, useState } from "react";
-import useOnclickOutside from "react-cool-onclickoutside";
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
```

Comment:
Are these patches really necessary?

Reply:
me parece que fue porque daba error al importar Beamer
<img width="538" alt="Captura de Pantalla 2022-12-19 a la(s) 8 42 45 a  m" src="https://user-images.githubusercontent.com/84402180/208439159-5c562341-f82c-453d-9aad-1e54cb623c2d.png">

sabes si hay una mejor forma de hacerlo?

Reply:
Did you try adding a reference to `@types/react-beamer`? does this library exist?

Reply:
If the typings do not exist for this library, you can declare the type in the project. It is explained in the first part of [this article](https://pjausovec.medium.com/how-to-fix-error-ts7016-could-not-find-a-declaration-file-for-module-xyz-has-an-any-type-ecab588800a8).

The advantage of doing this is that we can improve the typings when needed.

Reply:
Yes I tried, but give me an error: Not found

Reply:
> Yes I tried, but give me an error: Not found

You are right because there are no defined types for `react-beamer`: https://github.com/search?q=repo%3ADefinitelyTyped%2FDefinitelyTyped+%22react-beamer%22&type=code

Maybe the other solution could work (https://github.com/FromDoppler/doppler-menu-mfe/pull/234#discussion_r1052221998), in my opinion, it is slightly better, but if it does not work, the patch is ok.